recipes/pyomeca/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "pyomeca" %}
+{% set version = "2019.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ee89099c0250e8d41984581418bef2f1e9edb9cfe6ec42753b10612ea6c29f73
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
+    - pyyaml
+  host:
+    - python
```

If you need to constrain the Python version you can use this with `noarch` packages:
```suggestion
    - python >=3
```
(edit as appropriate and include the same thing in `run`)